internal/git/baseline_test.go

```
@@ -125,6 +127,56 @@ func TestCheckoutBaseline_ExtractsFiles(t *testing.T) {
+	gitRun(t, dir, "config", "user.email", "test@test.com")
+	gitRun(t, dir, "config", "user.name", "Test")
+
+	// Base commit: legacy teams/ layout.
+	os.MkdirAll(filepath.Join(dir, "teams"), 0o755)
+	os.WriteFile(filepath.Join(dir, "teams", "test.yml"), []byte("name: Test\n"), 0o644)
+	gitRun(t, dir, "add", "-A")
+	gitRun(t, dir, "commit", "-m", "init")
+	baseSHA := gitOutput(t, dir, "rev-parse", "HEAD")
+
+	// Worktree migrates teams/ -> fleets/ (rename shows as delete + add).
+	os.Rename(filepath.Join(dir, "teams"), filepath.Join(dir, "fleets"))
```

golangci-lint (2.12.2)

[error] Error return value of `os.Rename` is not checked

(errcheck)